packages/osd-stylelint-config/config/global_selectors.json

```diff
@@ -0,0 +1,30 @@
+{
+  "#opensearch-dashboards-body": {
+    "approved": [
+      "src/core/public/rendering/_base.scss"
+    ]
+  },
+  ".app-wrapper": {
+    "approved": [
+      "src/core/public/rendering/_base.scss"
+    ]
+  },
+  "/\\.[eo]ui/": {
```

Comment:
I think we shouldn't match the start of the string because there are cases like `&--selectableWrapper .euiSelectableList` where it isn't but should still be caught. And can confirm this is the case, however we can add the `i` flag just to be sure.